### PR TITLE
Feat/237 homebackground

### PR DESCRIPTION
**Pull request by sstraatemans**
## Reason:

Closes [issue link]

## Check off the following:

- [ ] I have reviewed my changes and run the appropriate tests.
- [ ] I have have run `rush change` to add the appropriate change logs.
- [ ] I have added/edited docs.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

File: packages/apps/kadena-docs/src/pages/index.tsx [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
 
-export default function Home(): JSX.Element {
+const Home: IDocsPageFC = () => {
   return (
-    <Stack>
-      <div>1</div>
-      <div>2</div>
-      <div>3</div>
-    </Stack>
+    <>
```
Comment (isaozler) [inline packages/apps/kadena-docs/src/pages/index.tsx:8]: Is this Fragment a requirement of MarcDoc or just a “left-over”?

**Review by isaozler — APPROVED** [review]

File: packages/apps/kadena-docs/src/pages/index.tsx (hunk 1 above)
Comment (sstraatemans) [reply packages/apps/kadena-docs/src/pages/index.tsx:8]: will remove

File: packages/apps/kadena-docs/src/components/Layout/components/Aside/AsideStyles.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    inset: 0,
+    backgroundColor: '$backgroundOverlayColor',
+    zIndex: 1,
+  },
+});
+
+export const CodeBackground: StyledComponent<typeof BaseBackground> = styled(
+  BaseBackground,
+  {
+    '--shadowWidth': '$sizes$20',
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/components/Aside/AsideStyles.ts:41]: I recommend using `$$shadowWidth` for locally scoped tokens 
https://stitches.dev/docs/styling#locally-scoped-tokens

File: packages/apps/kadena-docs/src/components/Layout/components/TitleHeader/style.ts [hunk 3]
```diff
@@ -1,40 +1,68 @@
...
-  backgroundPositionX: 'center',
-  backgroundPositionY: '90%',
-  transform: 'scaleX(-1)',
+
+export const HeaderWrapper: StyledComponent<'div'> = styled('div', {
+  gridArea: 'pageheader',
+  position: 'relative',
+  display: 'grid',
+  height: '300px',
+  gridTemplateRows: '260px 40px',
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/components/TitleHeader/style.ts:10]: Can these use the sizing tokens?

File: packages/apps/kadena-docs/src/components/Layout/components/styles.ts [hunk 4]
```diff
@@ -9,79 +9,84 @@ export const Wrapper: StyledComponent<'div'> = styled('div', {
...
   'div',
-  { layout?: 'landing' | 'normal' | undefined }
+  { layout?: 'landing' | 'normal' | 'code' | undefined }
 > = styled('div', {
+  '--asideMenuWidthMDDefault': '200px',
+  '--asideMenuWidthLGDefault': '300px',
+  '--asideMenuWidthMDCode': '300px',
+  '--asideMenuWidthLGCode': '400px',
+  '--asideMenuWidthXLCode': '500px',
+
```
Comment (eileenmguo) [inline packages/apps/kadena-docs/src/components/Layout/components/styles.ts:19]: I would also use the `$$` annotation here

**Review by eileenmguo — APPROVED** [review]

**Pull request merged**